install.php

```diff
@@ -1610,38 +1633,70 @@ function BuildFileList(){
 <a href="?dept&preflight-ok"><li>Departments</li></a>
 <a href="?dc&preflight-ok"><li>Data Centers</li></a>
 <a href="?cab&preflight-ok"><li>Cabinets</li></a>
-<a><li class="active">LDAP</li></a>
+<?php if ( AUTHENTICATION == "LDAP" ) echo '<a><li class="active">LDAP</li></a>'; ?>
+<?php if ( AUTHENTICATION == "AD" ) echo '<a><li class="active">AD</li></a>'; ?>
 <?php if(isset($complete)){ echo '<a href="?complete&preflight-ok"><li>Complete</li></a>'; }?>
 </ul>
 </div>
 <div class='main'>
 <h2><?php echo $config->ParameterArray['OrgName']; ?></h2>
-<h3>Installation Complete</h3>
-<?php echo $nodccab; ?>
+<h3>Authentication and Authorization</h3>
+<?php
+if (!isset($_SESSION['ldapauthset'])) {
```

Comment:
If they haven't done so, remind the user to press the Set button for LDAP/AD config.